scripts/helpers/openai-compatible-agent.py

```
@@ -152,9 +160,12 @@ def api_call(base_url, key, model, headers_extra, messages, max_tokens=0, reques
     headers = {"Authorization": "Bearer " + key, "Content-Type": "application/json", **headers_extra}
     body = json.dumps(payload).encode()
     endpoint = base_url.rstrip("/") + "/chat/completions"
-    scheme = urllib.parse.urlparse(endpoint).scheme
+    parsed_endpoint = urllib.parse.urlparse(endpoint)
+    scheme = parsed_endpoint.scheme
     if scheme not in {"http", "https"}:
         raise ValueError(f"unsupported OPENAI-compatible base URL scheme: {scheme or '<missing>'}")
```

Ruff (0.16.3)

[warning] Avoid specifying long messages outside the exception class

(TRY003)